AirCasting/SessionsSynchronization/Database/DatabaseMeasurement.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+extension Database {
+    public struct Measurement {
+        public let id: Int
```

Comment:
typealias for ID type

Reply:
👌
Note: currently there are some universal, app-wide structures that live inside the `CoreData` group like `SessionUUID`, `MeasurementID` etc.
I think this needs to be moved to some other place, because it's not only CoreData layer that uses it, it's all over the app (very versatile objects that are used by all app layers)